src/index.js

```diff
@@ -0,0 +1,36 @@
+import Weather from './weather';
+import Storage from './storage';
+import Dom from './dom';
+
+const storage = new Storage();
+
+const dom = new Dom();
+
+const storedWeather = storage.getLocationData();
+
+const weather = new Weather(storedWeather.city, storedWeather.unit);
+
+const changeBtn = document.getElementById('w-change-btn');
+const city = document.getElementById('city');
+const unit = document.getElementById('unit');
+
+function getWeather() {
+  weather.getWeather(city.value, unit.value)
+    .then(results => {
+      dom.updateDom(results);
+    })
+    // eslint-disable-next-line no-alert
+    .catch(err => alert(err));
```

Comment:
Opps! This should be production code, we shouldn't have `alert`, please remove the `alert`.
we could create the error message using a div.